Warmup-1/hasTeen/src/index.ts

```diff
@@ -0,0 +1,8 @@
+export function hasTeen(myArray: number[]): boolean {
+  for (let number of myArray) {
+    if (19 >= number && number >= 13) {
```

Comment:
Best practice is usually to have your variable first and then what you're comparing it to: `number <= 19`